### PR TITLE
windows: Add RBAC for EKS quickstart

### PR DESCRIPTION
**Pull request by lmm**
## Description

This PR adds some instructions to guide users to test out a Calico for Windows install on EKS more seamlessly.

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: getting-started/windows-calico/quickstart.md [hunk 1]
```diff
@@ -29,8 +29,23 @@ Whether you use etcd or Kubernetes datastore (kdd), the datastore for the Window
 - Remote access to the Windows node via Remote Desktop Protocol (RDP) or Windows Remote Management (WinRM)
 - Be able to run a command as Administrator using powershell.
 - Additionally, for EKS:
-    - The VPC controllers must be installed be installed to run Windows pods.
-    - The Windows instance role must have access to `secrets` in the calico-system namespace or kube-system namespace if you are using a non operator-managed Calico installation.
+  - The VPC controllers must be installed be installed to run Windows pods.
+  - The Windows instance role must have permissions to get `namespaces` and get `secrets` in the calico-system namespace (or kube-system namespace if you are using a non operator-managed {{site.prodname}} installation.)
+    - For non-production usage, run these commands below to install the permissions needed to install {{site.prodnameWindows}}.
```
Comment (song-jiang) [inline getting-started/windows-calico/quickstart.md:34]: Why `For non-production usage`? We've updated installation script to enable WinDSR by default so we should claim EKS support production ready.

**Review by song-jiang — COMMENTED** [review]
Just a comment

File: getting-started/windows-calico/quickstart.md (hunk 1 above)
Comment (lmm) [reply getting-started/windows-calico/quickstart.md:34]: I was just thinking that maybe there is a better way of doing this. I'll reword that.
I also need to replace the usage of the `view` role with something just to calico or kube-system get/list namespaces.

File: getting-started/windows-calico/quickstart.md [hunk 2]
```diff
@@ -29,8 +29,25 @@ Whether you use etcd or Kubernetes datastore (kdd), the datastore for the Window
 - Remote access to the Windows node via Remote Desktop Protocol (RDP) or Windows Remote Management (WinRM)
 - Be able to run a command as Administrator using powershell.
 - Additionally, for EKS:
-    - The VPC controllers must be installed be installed to run Windows pods.
-    - The Windows instance role must have access to `secrets` in the calico-system namespace or kube-system namespace if you are using a non operator-managed Calico installation.
+  - The VPC controllers must be installed be installed to run Windows pods.
```
Comment (song-jiang) [inline getting-started/windows-calico/quickstart.md:32]: remove one `be installed`

**Review by song-jiang — CHANGES_REQUESTED** [review]
one typo

**Review by song-jiang — APPROVED** [review]

**Pull request merged**